lib/ansible/modules/clustering/kubevirt/kubevirt_service.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kubevirt_service
+
+short_description: Manage Services for KubeVirt virtual machines
+
+description:
+    - Use Openshift Python SDK to manage Services for KubeVirt virtual machines.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    state:
+        description:
+            - Determines if an object should be created, patched, or deleted. When set to C(present), an object will be created, if it does not already exist. If set to C(absent), an existing object will be deleted. If set to C(present), an existing object will be patched, if its attributes differ from those specified.
+        default: present
+        choices:
+            - present
+            - absent
+        type: str
+    name:
+        description:
+            - Use to specify a Service object name.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Use to specify a Service object namespace.
+        required: true
+        type: str
+    selector:
+        description:
+            - TODO
+        type: dict
```

Comment:
Are we going to support k8s semantics for selectors here?

Reply:
Yup, no other way to do it.